bigbio/biodatasets/medical_data/medical_data.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List, Tuple, Dict
+import pandas as pd
+import datasets
+from bigbio.utils import schemas
+from bigbio.utils.configs import BigBioConfig
+from bigbio.utils.constants import Tasks
+
```

Comment:
well need a `_LOCAL=True` here .. we haven't added it to the contributing docs yet but all the datasets have it now

Reply:
Done

Reply:
@galtay is `_LOCAL` the last change this PR needs to be consistent with the latest changes? this PR looks good to me on the split side. It'll be great if you could confirm and then we can merge this one in.